indico/modules/rb/client/js/common/linking/LinkBar.jsx

```diff
@@ -33,16 +34,16 @@ const LinkBar = ({visible, clear, data}) => {
   if (!visible) {
     return null;
   }
-
-  const {type, title, eventURL, eventTitle} = data;
+  const fmtDate = date => moment(date).format('Do MMMM ha');
```

Comment:
`ha` uses am/pm which is not suitable for certain locales - better to use a locale-ware format string

Reply:
Just changed it to use 24h format.

<img width="532" alt="Screenshot 2024-05-27 at 08 16 23" src="https://github.com/indico/indico/assets/2722756/72b6ff52-413f-49bd-8c13-45df29b149a5">

Reply:
Dates are tricky :sweat_smile: In general, neither 12 or 24 hour is correct for all locales (`en_US` for example prefers 12-hour format). You should be use `Intl.DateTimeFormat` to get the right format for the current locale. Just keep in mind that the territory/language separator differs from the locales we use internally. You'll need to use `locale.replace('_', '-')` before passing the locale to `Intl.DateTimeFormat` (search the codebase for more examples). Since we are formatting a date range, I recommend also having a look at [`formatRange`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/formatRange) since different locales format date ranges differently

Reply:
Date rendering being tricky, it may be worth adding a helper function in the project so that datetime ranges are rendered canonically.

Reply:
Yep that should be fine :) I just added missing time localization (see https://github.com/indico/indico/pull/6362)